pic-sure-auth-services/src/main/java/edu/harvard/hms/dbmi/avillach/auth/service/auth/FENCEAuthenticationService.java

```diff
@@ -514,11 +514,18 @@ private Privilege upsertClinicalPrivilege(String studyIdentifier, String project
                     +"\":[\""
                     +studyIdentifierField
                     +"\"]},"
-                    +"\"numericFilters\":{},\"requiredFields\":[],"
-                    +"\"fields\":[\"" + parentAccessionField + "\"],"
-                    +"\"variantInfoFilters\":[{\"categoryVariantInfoFilters\":{},\"numericVariantInfoFilters\":{}}],"
+                    +"\"numericFilters\":{},\"requiredFields\":[],";
+
+                    if(JAXRSConfiguration.idp_provider.equalsIgnoreCase("fence")) {
```

Comment:
No change needed, but it's nice to put the string constant as the first term in a .equals call to guarantee no NPE.

Reply:
```suggestion
                    if("fence".equalsIgnoreCase(JAXRSConfiguration.idp_provider)) {
```

Reply:
@Luke-Sikina Updated to avoid NPE